gio/src/unix_socket_address.rs

```diff
@@ -1,10 +1,10 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+use crate::prelude::*;
 use crate::SocketAddress;
 use crate::UnixSocketAddress;
-use crate::UnixSocketAddressExt;
 use crate::UnixSocketAddressType;
-use glib::object::{Cast, IsA};
+use glib::prelude::*;
```

Comment:
i believe gio prelude contains glib's prelude already no?

Reply:
Indeed, removed here and in a few other places.